File: src/net/merayen/elastic/uinodes/list/signalgenerator_1/Info.java
package net.merayen.elastic.uinodes.list.signalgenerator_1;

import net.merayen.elastic.uinodes.BaseInfo;

public class Info implements BaseInfo {
	@Override
	public String getName() {
		return "Signal generator";
	}

	@Override
	public String getDescription() {
		return "Generates audio signal";
	}

	@Override
	public String[] getCategories() {
		return new String[]{"Generators"};
	}
}

File: src/main/kotlin/net/merayen/elastic/uinodes/list/histogram_1/Info.kt
package net.merayen.elastic.uinodes.list.histogram_1

import net.merayen.elastic.uinodes.BaseInfo

class Info : BaseInfo {
	override fun getName() = "Histogram"
	override fun getDescription() = "A histogram of the data/audio"
	override fun getCategories() = arrayOf("Analyze")
}